Formula/meta-package-manager.rb

```diff
@@ -0,0 +1,417 @@
+class MetaPackageManager < Formula
+  include Language::Python::Virtualenv
+
+  desc "🎁 wraps all package managers with a unifying CLI"
```

Comment:
Though less fun, this is probably easier to understand:

```suggestion
  desc "Wrapper around all package managers with a unifying CLI"
```

Reply:
Fair enough! I understand it's good to not be too fancy here.